org.andrew.ytdownloader.yml

```diff
@@ -0,0 +1,53 @@
+app-id: org.andrew.ytdownloader
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=home
+  - --talk-name=org.freedesktop.Notifications
+  - --share=network
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: ytdownloader
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/ytdownloader/flatpak-node/cache
+        npm_config_cache: /run/build/ytdownloader/flatpak-node/npm-cache
+        npm_config_offline: 'true'
+    build-commands:
+      # Install npm dependencies
+      - npm install --offline
+      # Build the app; in this example the `dist` script
+      # in package.json runs electron-builder
+      - |
+        . ../flatpak-node/electron-builder-arch-args.sh
+        npm run dist -- $ELECTRON_BUILDER_ARCH_ARGS  --linux --dir
+      # Bundle app and dependencies
+      - cp -a dist/linux*unpacked /app/main
+      # Install app wrapper
+      - install -Dm755 -t /app/bin/ ../run.sh
+    subdir: main
+    sources:
+      #- type: archive
+      #  url: https://github.com/aandrew-me/ytDownloader/archive/refs/tags/v1.7.0.tar.gz
+      - type: dir
+        path: ..
+        dest: main
```

Comment:
This needs to be an actual git repo or a tarball.

Reply:
Oh okay